test/fixtures/nested-default-route.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <single-spa-router mode="history" base="/">
```

Comment:
This is a good place to start when reviewing this PR. The `default` attribute on `<route>` elements is the new functionality.